Tests/NIOTests/BootstrapTest.swift

```diff
@@ -529,6 +529,25 @@ class BootstrapTest: XCTestCase {
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: elg))
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: el))
     }
+    
+    func testShorthandOptionsAreEquivalent() throws {
+        func bindAndGetReuseAddrOption(_ applyBootstrapOptions : (ServerBootstrap) -> ServerBootstrap) throws ->
+                ChannelOptions.Types.SocketOption.Value {
+            let bootstrap = applyBootstrapOptions(ServerBootstrap(group: group))
+            let serverChannel = try bootstrap.bind(host: "127.0.0.1", port: 0).wait()
+            let reuseValue = try serverChannel.getOption(ChannelOptions.socketOption(.reuseaddr)).wait()
+            try serverChannel.close().wait()
+            return reuseValue
+        }
+        
+        let sbLongReuseValue = try bindAndGetReuseAddrOption { bs in
+            bs.serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: 1) }
+        let sbShortReuseValue = try bindAndGetReuseAddrOption { bs in bs.serverOptions([.reuseAddr])}
+        let sbNoReuseValue = try bindAndGetReuseAddrOption { bs in bs }
```

Comment:
I *think* standard Swift syntax just calls for the use of `$0`

```suggestion
        let sbNoReuseValue = try bindAndGetReuseAddrOption { $0 }
```

Reply:
Taken your change.